agb-image-converter/src/lib.rs

```diff
@@ -310,6 +314,7 @@ pub fn include_aseprite_inner(input: TokenStream) -> TokenStream {
     let filenames: Vec<PathBuf> = parsed
         .iter()
         .map(|s| s.value())
+        .map(|s| s.replace(OUT_DIR_TOKEN, &out_dir_path))
         .map(|s| Path::new(&root).join(&*s))
```

Comment:
is `$OUT_DIR` relative to the cargo manifest dir or is it absolute?

Reply:
The value you get from std::env::var("OUT_DIR") is absolute.

Reply:
Ah, missed this bit in the docs:
> If path is absolute, it replaces the current path.